docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -100,6 +100,42 @@ Apart from `extra_links.tmpl` and `extra_tabs.tmpl`, there are other useful temp
 - `body_outer_post.tmpl`, before the bottom `<footer>` element.
 - `footer.tmpl`, right before the end of the `<body>` tag, a good place for additional Javascript.
 
+#### Example: Mermaid.js
+
+If you would like to add [mermaid.js](https://mermaid-js.github.io/) support to Gitea's markdown you simply add:
+
+```html
+<script src="https://unpkg.com/mermaid@8.4.5/dist/mermaid.min.js"></script>
+<!-- or wherever you have placed it -->
+<script>mermaid.init(".language-mermaid")</script>
+```
+
+to `footer.tmpl` within the `{{if .RequireHighlightJS}}...{{end}}` block
```

Comment:
It would be better with a tmpl within the `custom` folder.

Reply:
this is?

Reply:
If the site admin has to place this code within `base/footer.tmpl` it would require to be updated in case of other changes. Better to have a specific custom file for it.

Reply:
Ok it turns out that it being in custom/footer.tmpl doesn't break anything. It just means that it needs to be loaded everytime I guess we could stick a `{{if .RequirHighlightJS}}....{{end}}` around it?